devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy

```diff
@@ -17,7 +17,7 @@
 
 import java.text.SimpleDateFormat
 
-def devVer() { return "2.0.0" }
+def devVer() { return "3.0.0" }
```

Comment:
why this change?

Reply:
Just for reference. Do you want me to return it?

Reply:
changing this causes all the DTH to re-initialize. See `generateEvent` method